<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>498</width>
    <height>227</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>-50</x>
      <y>20</y>
      <width>911</width>
      <height>531</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>6</number>
    </property>
    <widget class="QWidget" name="pageWelcome">
     <widget class="QPushButton" name="btnStart">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>60</y>
        <width>91</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnExit">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>60</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>10</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Math World</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="pageMainMenu">
     <widget class="QPushButton" name="btnQuestion">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>50</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Questions</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnFormulas">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>40</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Formulas</string>
      </property>
     </widget>
     <widget class="QPushButton" name="BtnMainBack">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>100</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="pageTopicSelect">
     <widget class="QPushButton" name="btnIntegralQuestion">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>30</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Integral</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnDerivativeQuestion">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>20</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Derivative</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnLimitQuestion">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>100</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Limit</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnTopicBack">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>100</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="pageQuestion">
     <widget class="QPushButton" name="btnSubmit">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>130</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Submit</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnQuestionBack">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>130</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>80</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>10</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>191</width>
        <height>41</height>
       </size>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="pageFeedback">
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>10</y>
        <width>251</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>70</y>
        <width>261</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnFeedBackBack">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>130</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnNextQuestion">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>130</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Next</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="pageFormulaSelect">
     <widget class="QPushButton" name="btnIntegralFormulas">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>20</y>
        <width>71</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Integral</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnLimitFormulas">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>100</y>
        <width>71</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Limit</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnDerivativeFormulas">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>20</y>
        <width>71</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Derivative</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnFormulaBack">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>110</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="pageExitScreen">
     <widget class="QTextBrowser" name="FormulaBrowser">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>0</y>
        <width>256</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnExitBack">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>30</y>
        <width>56</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnExitScreen">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>80</y>
        <width>56</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>498</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
